### ISSUE TITLE
Multiple Keyword searches (comparison)

### ISSUE THREAD

**Issue by Dayjo** [pat310/google-trends-api#53]
Hello,

Since the version change (and the documentation change), It's unclear how to query multiple keywords. Previously it was a keywords parameter which could be an array. Now it's singular `keyword`, and it doesn't seem to support an array.

I have tried using a comma separated list but just get empty results, i.e. 
```json
{"default":{"timelineData":[],"averages":[]}}
``` 

I tried the following;
```js
googleTrends.interestOverTime({keyword: 'toast, strawberry'})
.then(function(results){
        console.log('These results are awesome', results);
        callback(null, results);
})
.catch(function(err){
        callback(err, 'Oh no there was an error');
});
```

also tried..

```js
googleTrends.interestOverTime({keyword: ['toast', 'strawberry']})
```

Essentially I'm looking for something like this;

![image](https://cloud.githubusercontent.com/assets/1255512/23414545/37e2c4ca-fdd4-11e6-915f-50050e8a9bbf.png)

Have I missed something?

**Comment by pat310** [comment]
@Dayjo I left implementation of searching multiple keywords out, but it's something that could be added.  Alternatively, you could make an array of promises, something like the following:
```js
let words = ['toast', 'strawberry'];
let promiseArr = words.map((keyword) => {
 return googleTrends.interestOverTime({keyword}); 
});

Promise.all(promiseArr)
.then((results) => {
 console.log('results!', results); 
/* 
 * here results is an array where the first element are the results for `toast`
 * and the second element are the results for `strawberry`
*/ 
});
```

**Comment by Dayjo** [comment]
@pat310 Unfortunately that doesn't have the same effect as the scale is always relative to the query. 

i.e. the individual query for strawberry maxes at 100 (the highest interest), and the individual query for toast also maxes at 100. If you query them together, however, they scale relative to each other so that the max for either is 100 and actually the max for toast is much lower (around 50, so half the interest of strawberry). 

I hope that makes sense haha

TL;DR - You cannot compare two individual queries as the scale is relative to itself, not each other.

**Comment by Dayjo** [comment]
I have determined what's needed here. The `comparisonItem` simply needs to send multiple objects. Should be straight forward enough to add support for multiple objects I hope! :)
